### PR TITLE
Clarifications around BLOCK_MAX_BYTES_MAX_FRACTION

### PR DESCRIPTION
**Pull request by adizere**
I was documenting the use and requirements around Hermes health check and RPC endpoints, and took the occasion to update some sub-par doc.

## Description

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules).
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: crates/relayer/src/chain/cosmos.rs [hunk 1]
```diff
@@ -122,9 +122,18 @@ pub mod types;
 pub mod version;
 pub mod wait;
 
-/// fraction of the maximum block size defined in the Tendermint core consensus parameters.
-pub const GENESIS_MAX_BYTES_MAX_FRACTION: f64 = 0.9;
-// https://github.com/cosmos/cosmos-sdk/blob/v0.44.0/types/errors/errors.go#L115-L117
+/// Defines an upper limit on how large any transaction can be.
+/// This upper limit is defined as a fraction relative to the block
```
Comment (seanchen1991) [inline crates/relayer/src/chain/cosmos.rs:126]:
```suggestion
/// This upper limit is defined as a fraction relative to the block's
```

File: crates/relayer/src/chain/cosmos.rs [hunk 2]
```diff
@@ -122,9 +122,18 @@ pub mod types;
...
+/// maximum bytes. For example if the fraction is `0.9`, then
+/// `max_tx_size` will not be allowed to exceed 0.9 of the
+/// maximum block size of any Cosmos SDK network.
+///
+/// The [default max. block size in Tendermint 0.37 is 21MB](tm-37-max).
+/// With a fraction of `0.9`, then Hermes will never permit the configuration
+/// of `max_tx_size` to exceed ~18.9MB.
+///
+/// [tm-37-max]: https://github.com/tendermint/tendermint/blob/v0.37.0-rc1/types/params.go#L79
+pub const BLOCK_MAX_BYTES_MAX_FRACTION: f64 = 0.9;
```
Comment (seanchen1991) [inline crates/relayer/src/chain/cosmos.rs:136]: Could we add some rationale for why 0.9 is the default value?

Comment (adizere) [reply crates/relayer/src/chain/cosmos.rs:139]: d56adcf369d96ce1e4d1e8194d221aceda26f750

**Review by seanchen1991 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**